deployments/examples/ocis_collaboraton/README.md

```diff
@@ -0,0 +1,26 @@
+# Documentation
+The documentation is incomplete because the Collaboration server is in a development state.
+
+# Infinite Scale Collaboration Deployment Example
+
+This deployment example of the oCIS with the new Collaboration server.
```

Comment:
```suggestion
# Infinite Scale Collaboration Deployment Example

IMPORTANT: This documentation is a first draft because the Collaboration service is currently under development.

This oCIS deployment example shows the setup with the new Collaboration service replacing on a long run the WOPI deployment example.
```